group/init/README.md

```diff
@@ -42,8 +42,9 @@ var prevMsgId = 'ssb:message/classic/Zz-Inkte70Qz1UVKUHIhOgo16Oj_n37PfgmIzLDBgZw
 var feed_id = ... BFE binary encoding of feed_id
 var prev_msg_id = ... BFE binary encoding of feed_id
 
-var group_key = ... symetric key as buffer
-vat msg_key = ... make up a one use key for the msg
+var group_key = ... symmetric key as buffer
+var own_key = ... symmetric key as buffer
+var msg_key = ... make up a one use key for the msg
```

Comment:
:fire: this needs to be changed to clarify:

```js
var group_key = { key, scheme: 'xxxxx' }      // key is symmetric key as buffer
var own_key = { key, scheme: 'yyyy' }          // key is a symmetric key as a buffer
var msg_key =  ... make up a one use key for the msg as a buffer
```
put in actual strings for xxxxx, yyyyy
And check what I'm saying makes sense in the code

Reply:
envelopejs

Reply:
i have no idea how `own_key` works and it's not really documented, how would i explain it to someone else lol? Different PR?

Reply:
It's here https://github.com/ssbc/private-group-spec/tree/master/direct-messages#b-self-case---mapping-our-own-feed_id-to-recp_key

Could be in a clearer space but some of these ideas have evolved as we've used them and talked about them.

Reply:
Ah thanks it actually does mention the scheme, I'll fix.

Two issues with that section though https://github.com/ssbc/private-group-spec/issues/28 https://github.com/ssbc/private-group-spec/issues/29

Reply:
i think i fixed this now